docs/site/Examples.md

```diff
@@ -14,6 +14,10 @@ LoopBack 4 comes with the following example projects:
 - **[rpc-server](https://github.com/strongloop/loopback-next/tree/master/examples/rpc-server)**:
   An example showing how to implement a made-up RPC protocol.
 
+- **[loopback4-example-shopping](https://github.com/strongloop/loopback4-example-shopping)**:
+  An online e-commerce demo to validate / test the LoopBack 4 framework
+  readiness
```

Comment:
Add a trailing `.`?

Reply:
Since you are here, can you also add the following?

- todo: Tutorial example on how to build an application with LoopBack 4.
- todo-list: Continuation of the todo example using relations in LoopBack 4.
- hello-world: A simple hello-world Application using LoopBack 4.
- log-extension: An example extension project for LoopBack 4.
- rpc-server: A basic RPC server using a made-up protocol.
- soap-calculator: An example on how to integrate SOAP web services.
- express-composition: A simple Express application that uses LoopBack 4 REST API.
```

Reply:
@raymondfeng Most of them are added as tutorial, see:

https://loopback.io/doc/en/lb4/Tutorials.html:

- todo
- todo-list
- log-extension
- soap-calculator
- express-composition

And the other 2 examples:

- hello-word
- rpc-server

are added already, just a few lines above my new doc.

Reply:
Oh.